### ISSUE TITLE
Markdown parsing issue 

### ISSUE THREAD

**Issue by rahul286** [EasyEngine/docs#48]
please sees screenshot here https://twitter.com/brianjking/status/1067864888041566208

It appears that markdown parser is not working completely or isnt compatabile with Gutenberg editor. 

We likely need to fix this on WordPress end.

**Comment by rahul286** [comment]
@rahulsprajapati we might need to fix https://github.com/mAAdhaTTah/wordpress-github-sync if it doesn't support Gutenberg. 

I am not sure how Gutenberg's HTML comment will work with Markdown.

So for time being, we might need to disable Gutenberg on `handbook` CPT. We can use https://wordpress.org/plugins/gutenberg-ramp/ for this part (if needed).

**Comment by brianjking** [comment]
@rahul286 Are you indicating that the docs should in fact be standard markdown that is then parsed and generated into WordPress posts/pages with appropriate formatting? If so, this is great news.

**Comment by rahul286** [comment]
@brianjking Yep. The intention is to not only use plain & simple markdown but also use Jekyll and front-matter https://jekyllrb.com/docs/front-matter/

Now the WordPress is used as a presentation layer. This way we don't need to design or maintain a separate Jekyll theme.

Besides, we can use some WordPress features such as SEO plugins, related posts/docs, and more. 

With WordPress, we will be able to allow logged in users to subscribe to pages they are interested in and receive changes to those pages via email!

Now, we started with a good plugin https://github.com/mAAdhaTTah/wordpress-github-sync which takes care of data sync part between markdown files here and WordPress post content nicely. It's just the plugin we depend on isn't tested with Gutenberg.

Our devs couldn't fix yet as we are clueless as of how to preserve Gutenberg comments on markdown sync. But if that turns out to be a blocker, we will remove Gutenberg for the documentation part. 

Documentation being simple pages, may not require Gutenberg.
